Source/Renderer/UniformState.js

```diff
@@ -885,10 +906,7 @@ define([
         this._resolutionScale = canvas.width / canvas.clientWidth;
 
         var camera = frameState.camera;
-
-        setView(this, camera.viewMatrix);
-        setInverseView(this, camera.inverseViewMatrix);
-        setCamera(this, camera);
+        this.updateCamera(camera);
```

Comment:
Removing `this.updateCamera(camera);` breaks a lot of tests, so I'm leaving it in.